src/Umbraco.Web/HealthCheck/Checks/Config/ProvidedValueValidation.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Umbraco.Web.HealthCheck.Checks.Config
+{
+    public enum ProvidedValueValidation
```

Comment:
I think we should give these enums specific values since inevitably it seems like more might be added in the future

Reply:
Hi @Shazwazza, can you expand on what you mean by giving these enums specific values?

Thanks

Reply:
Like `None = 1, Email = 2,` etc...

Reply:
Is it because other enums may be added over time in between existing enum values and that can change the underlying assigned value? Not sure the code relies on the underlying values at all though?

Reply:
Yes that's what i mentioned above. Code does rely on these values (in IL these ARE the values) so if you add one on the top, the values change, then you can't just drop in a DLL replacement because the manifest of this enum has changed, you'd need to recompile which we don't want. Always advised to add real values to enums due to this

Reply:
Ah, interesting. Thanks for explaining. I thought it would only be a problem if the enum values were being stored elsewhere. @AndyButland would you like to make that change?

I've got another PR I'm working on that needs this merged in first.

Reply:
Sure.  Though having pushed the change to my branch - see [here](https://github.com/AndyButland/Umbraco-CMS/commits/u4-10004) - it's not updating this PR.  Not sure why I'm afraid.

Reply:
Thanks @AndyButland. The commit seems to be showing on the PR for me. I am however seeing a merge conflict with /AbstractConfigCheck.cs. Does that need to be resolved first?

Reply:
Weird, I'm not seeing in in the commits to this PR.  But yes, I did this nearly two years ago so I suspect there's a few issues with merge conflicts.  I think that'll need to be resolved by the HQ devs if and when it's accepted and merged in.

Reply:
Makes sense. Thanks. @Shazwazza are you happy for this to be put on track for merging now?

Reply:
Have added to the backlog, hopefully can review/merge next sprint